docs/content/concepts/graphql-rpc.mdx

```diff
@@ -2,11 +2,12 @@
 title: Sui GraphQL RPC
 ---
 
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
-
 This section explains some of the common concepts when working with GraphQL, such as altering behavior using [HTTP headers](./graphql-rpc/headers), re-using query snippets with [variables](./graphql-rpc/variables) and [fragments](./graphql-rpc/fragments), consuming [paginated queries](./graphql-rpc/pagination), and understanding and working within the [limits](./graphql-rpc/limits) enforced by the service.
 
 For more details on GraphQL fundamentals, see the introductory documentation from [GraphQL](https://graphql.org/learn/) and [GitHub](https://docs.github.com/en/graphql/guides/introduction-to-graphql).
 
-If you are interested in how to interact with the Sui network via the GraphQL RPC, see the [getting started](guides/developer/getting-started/graphql-rpc.mdx) guide, and the [full API reference](/references/sui-graphql). If you are looking for information on migrating from JSON-RPC to GraphQL, see the [migration](../guides/developer/advanced/graphql-migration.mdx) guide.
+## Related links
+
+- [Sui GraphQL Reference](../references/sui-graphql.mdx): Auto-generated reference for the Sui GraphQL API. 
+- [Migrating to GraphQL](../guides/developer/advanced/graphql-migration.mdx): Migrate from JSON-RPC API to GraphQL. 
+- [Querying Sui GraphQL](../guides/developer/getting-started/graphql-rpc.mdx): Get started making queries of Sui GraphQL. 
```

Comment:
nit: We should order this link before the migrating one given that it's the getting started guide.